app/helpers/green_lanes_helper.rb

```diff
@@ -54,6 +54,14 @@ def render_exemptions(assessments, category)
     end
   end
 
+  def yes_no_options
+    [OpenStruct.new(id: 'yes', name: 'Yes'), OpenStruct.new(id: 'no', name: 'No')]
```

Comment:
nice one :rocket: 

I was just thinking that Struct may be enough, we don't need to expand them with new keys

Reply:
Yeah possibly ive not tested to be honest, i cant remember why i used this pattern in the past

Reply:
I think these relate more to the form than the green lanes helper?